### ISSUE TITLE
AssertionError on multi-gpu system

### ISSUE THREAD

**Issue by mstud** [ahrnbom/strudl#1]
When starting the webserver via "python server.py" and then going to the webpage, we get AssertionErrors:
```  File "/code/server.py", line 336, in get_job_status
    cstatus = status()
  File "/code/compstatus.py", line 37, in status
    gpu_util, gpu_mem = gpu()
  File "/code/compstatus.py", line 21, in gpu
    assert(len(lines) == 2)
AssertionError
```

That's probably because we have multiple GPUs in our systems, so 
`nvidia-smi --query-gpu=utilization.gpu,memory.total,memory.used --format=csv,noheader,nounits`
outputs 4 rows instead of just one.
Not sure in how far multiple GPUs can be used by Strudl anyway, but our batch system sets CUDA_VISIBLE_DEVICES within job contexts, so this could possibly be fixed somewhat by just adding
`-i $CUDA_VISIBLE_DEVICES` to the parameters passed to the `nvidia-smi` call if this environment variable is set.

**Comment by ahrnbom** [comment]
Feel free to make a pull request with the fix you suggested. We don't have a multi-GPU environment to test this with, unfortunately. 

STRUDL doesn't have any code that explicitly utilizes multiple GPUs. All GPU code currently uses tensorflow so any environment variables that affect tensorflow can be used to control such things.

**Comment by Flamefire** [comment]
I ran into the same problem and appended the fix to my PR #6
